### ISSUE TITLE
Humans should be able to get other people's blood on them

### ISSUE THREAD

**Issue by 5hart** [studio-minus/ppg-bugs-and-requests#998]
**Feature description**
I've noticed that other human's blood, or any liquid for that won't show up on other humans. 

**Feature purpose**
Androids are able to show blood on them if they crush a poor soul, so why not people? I assumed this was so you don't get confused on whether or not it's THEIR blood, but I still think it's a nice thing to have, it can make stabbing or shooting someone more dramatic.

**Comment by wordpass1** [comment]
Zooi better add this. Rip and tear, Playground God.

**Comment by TheNoodle76** [comment]
yeah like you shoot somebody close and the blood splashes onto them. I think this is a great idea and it should be added

**Comment by 5hart** [comment]
Putting this out there again since a new update came out with new gore

**Comment by Drogo-man** [comment]
There is a mod for this.
Search "human decal enabler"
